### ISSUE TITLE
Steam Connection Issues

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#10232]
We got a problem of three streamers wanting to play the game, but one of them not being able to connect to the server another one hosts. The third one can.

They already checked firewall and game ports on both host and affected client with no luck.

Can anyone look into that and get me which information he needs?

They join each other through steam.

**Comment by mikiZmiki** [comment]
Manual connection gave the same results. when i host servers in other games everything work fine, maybe problem is in his configuration. Two of us are living in the same city, we have the same ISP, our iP is the same at the beginning only two last digits are different maybe this cause problems

**Comment by mirasrael** [comment]
Hi @mikiZmiki 
1. Do you have anti-virus or firewall enabled?
2. Do you have external IP address?

**Comment by mikiZmiki** [comment]
Hi,
I don't have anti-virus, firewall is enabled.

I have internal IP

**Comment by mirasrael** [comment]
Can we have a chat in Discord for simpler communication? My name `mirasrael` in Discord.

**Comment by mirasrael** [comment]
After research wasn't able to solve this problem.
Having Player A and Player B and me (C).
A, C can connect B.
A, B can connect C.
C can connect A.
B can't connect A.
We tried:
- everything with Firewall (reconfigure rules, disable it completely);
- A and B uses Hamachi (VPN for LAN over Internet), we tried with enabled/disabled Himachi in any combination, no success;
- tried direct connection, works for C to A, doesn't work for B to A;
- tried to bind different IP address (every network interface and 0.0.0.0). Was able to connect B to A with both Hamachi enabled and bind to Hamachi network interface, but in this case C can't connect to A.

In the end decided B will run a server, but we need other ideas how it can be fixed.
@johnkslg any thoughts?
